src/main/java/com/unascribed/fabrication/mixin/d_minor_mechanics/furnace_minecart_any_fuel/MixinFurnaceMinecartEntity.java

```diff
@@ -57,7 +57,7 @@ public void interact(PlayerEntity player, Hand hand, CallbackInfoReturnable<Acti
 	@Redirect(at=@At(value="INVOKE", target="net/minecraft/recipe/Ingredient.test(Lnet/minecraft/item/ItemStack;)Z"),
 			method="interact(Lnet/minecraft/entity/player/PlayerEntity;Lnet/minecraft/util/Hand;)Lnet/minecraft/util/ActionResult;")
 	public boolean testFuel(Ingredient subject, ItemStack stack) {
-		return MixinConfigPlugin.isEnabled("*.furnace_minecart_any_fuel") ? false : subject.test(stack);
+		return !MixinConfigPlugin.isEnabled("*.furnace_minecart_any_fuel") && subject.test(stack);
```

Comment:
Really, this should get blown up into two lines for clarity. Neither form of this is very clear.